packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -41,6 +41,8 @@ export const globalStyles = css`
     --wui-color-fg-100: #e4e7e7;
     --wui-color-fg-200: #949e9e;
     --wui-color-fg-300: #6e7777;
+
+    --wui-rgba-transparent: rgba(255, 255, 255, 0.2);
```

Comment:
Not too descriptive, perhaps something like `--wui-color-translucent` ? This one will also likely end up being used in few other components like toast notification perhaps.

Reply:
Renamed it to `wui-overlay-020`, because we will use this for more components.

Reply:
good naming idea 👍 
`--wui-color-overlay-20` then perhaps? Just to make it clearer